frontend/src/components/Common/Chart/index.tsx

```diff
@@ -208,8 +223,21 @@ const Chart = memo(
           fill: false,
         }));
       }
+      if (data.GoogleFloodConfig) {
```

Comment:
This PR is introducing a lot of extra complexity to this file. I'll open an issue recommending to split it by functions/services